.gitlab-ci.yml

```diff
@@ -1013,6 +1029,8 @@ workflow:
 
 .if_run_e2e_tests:
   - <<: *if_run_e2e
+  - <<: *if_disable_e2e
```

Comment:
Yes, but this is like that because most of the time the variable is not set, and this is considered the default. When setting the variable to true or false we can explicitly enable or explicitly disable the `e2e_tests`

Reply:
Would it be easier to give the variable a default value in the `variables` block? Also with regards to ordering, I feel like the disable rule should be checked before the run rule? It's a nitpick and I'm not 100% behind the thought, but it came to mind.

Reply:
Sure I could check the `disable_e2e` first. But it would not change the behaviour since the variable cannot match several conditions

Reply:
so we have 3 behaviours from a boolean variable, should we use `auto/manual/off` instead of `true/false` (maybe later)?

Reply:
Good point from @chouetz here - changing the values makes sense to _not_ be boolean if they're _not actually boolean_.

Reply:
That's indeed a good point. I will make the change here.

Reply:
I created the variable with `auto` `off` and `manual`. Default to `manual` the follow up thing I proposed ^ should be addressed later